src/mds/MDSMap.h

```diff
@@ -193,6 +193,7 @@ class MDSMap {
    */
 
   mds_rank_t max_mds; /* The maximum number of active MDSes. Also, the maximum rank. */
+  string balancer; /* The name and version of the metadata load balancer. */
```

Comment:
I guess it's the name and version if someone named their objects that way, but really this is just a rados object name, right?